app/Bus/Commands/Component/AddComponentCommand.php

```diff
@@ -69,12 +69,12 @@
      */
     public $rules = [
         'name'        => 'required|string',
-        'description' => 'string',
-        'status'      => 'int|min:1|max:4',
-        'link'        => 'url',
-        'order'       => 'int',
-        'group_id'    => 'int',
-        'enabled'     => 'bool',
+        'description' => 'required|string',
```

Comment:
This should be `nullable|string`, otherwise you're forced to provide a description for components when creating them through the API (even though you can have an empty description when creating or editing them in the dashboard).

Reply:
Hey! Feel free to submit a PR :)

The command is now called `CreateComponentCommand`.